code/modules/clothing/under/color.dm

```diff
@@ -33,12 +38,25 @@
 	item_color = "black"
 	resistance_flags = NONE
 
+/obj/item/clothing/under/color/jumpskirt/black
+	name = "black jumpskirt"
+	icon_state = "blackjumpskirt"
+	item_state = "bl_suit"
+	item_color = "blackskirt"
+	resistance_flags = NONE // I am going to assume this is here for a reason
```

Comment:
the black jumpsuit sets its resistance flags to none, I did it for the jumpskirt just in case lol

I don't think any of them should have resistance to anything

Reply:
Looks like the black jumpsuit doesnt catch fire because of it (lava floor forces it to), no idea why someone set it to this. Feel free to do whatever.

Reply:
ill leave it as is